sympy/physics/continuum_mechanics/beam.py

```diff
@@ -1461,7 +1461,7 @@ def shear_stress(self):
         """
         return self.shear_force()/self._area
 
-    def plot_shear_stress(self, subs=None):
+    def plot_shear_stress(self, subs=None, show=True):
```

Comment:
This parameter needs to be added to the docstrings of the functions. See the docstring style guide for the parameter list format.